app/models/mortality_tracking.rb

```diff
@@ -53,24 +53,11 @@ def self.create_from_csv_data(attrs)
     #
     # Note - DateTime.strptime will raise a ArgumentError
     # if there is a issue converting.
-    attrs['spawning_date'] = DateTime.strptime(attrs.delete('spawning_date'), '%m/%d/%y')
-    attrs['mortality_date'] = DateTime.strptime(attrs.delete('mortality_date'), '%m/%d/%y')
+    attrs['spawning_date'] = DateTime.strptime(attrs.delete('spawning_date'), '%m/%d/%y') rescue nil
+    attrs['mortality_date'] = DateTime.strptime(attrs.delete('mortality_date'), '%m/%d/%y') rescue nil
 
     new(attrs)
   end
 
-  validates_inclusion_of :raw, in: [true, false]
```

Comment:
@corneliusellen was the removing of these validations intentional? Looks like they're causing failures in specs (which is good!), so I'd just need to know whether we change the code or change the specs :)